src/DataValidation.py

```diff
@@ -188,10 +196,68 @@ def checkItemsThatShouldBeRequired():
                 if isinstance(region_requires, str):
                     if '|{}|'.format(item["name"]) in region_requires:
                         raise ValidationError("Item %s is required by region %s, but the item is not marked as progression." % (item["name"], region_name))
+                    if itemValues and 'ItemValue' in region_requires:
+                        for value in re.findall(r'\{ItemValue\(([^:]*)\:[^)]+\)\}', region_requires):
+                            if value.lower().strip() in [i.lower().strip() for i in item["value"].keys()]:
+                                raise ValidationError(f"Item {item['name']} has a value for the '{value}' value required by region '{region_name}', but the item is not marked as progression.")
+
                 else:
                     if item["name"] in region_requires:
                         raise ValidationError("Item %s is required by region %s, but the item is not marked as progression." % (item["name"], region_name))
 
+    @staticmethod
+    def checkIfEnoughItemsForValue():
+        values_available = {}
+        values_requested = {}
+
+        # get all the available values with total count
+        for item in DataValidation.item_table:

```

Comment:
Throw a `if progression or skip balancing` here instead of the above checks, so only progression items are counted towards this check.

Reply:
that a solution I didnt consider before checking if there is enough progression and the rest is just bonus value.